docs/Admin.md

```diff
@@ -658,7 +658,7 @@ React-admin uses [react-query](https://react-query-v3.tanstack.com/) to fetch, c
 * Query results that are no longer used in the current page are labeled as "inactive" and remain in the cache in case they are used again at a later time.
 * By default, "inactive" queries are garbage collected after 5 minutes.
 * Queries that fail are silently retried 3 times, with exponential backoff delay before capturing and displaying an error to the UI.
-* Query results by default are structurally shared to detect if data has actually changed and if not, the data reference remains unchanged to better help with value stabilization with regards to `useMemo` and `useCallback`. 
+* Query results by default are structurally shared to detect if data have actually changed and if not, the data reference remains unchanged to better help with value stabilization with regards to `useMemo` and `useCallback`. 
```

Comment:
Not sure about this one. To me, here we are talking about the JS Object named 'data' (which reference has to be stable across rerenders), and hence I'd keep it a singular.

Reply:
what about if the data object has ?

Reply:
On 2nd thought, let's go with your initial change. In the first part of the sentence, 'data' really means the data fetched from the API, and we are implicitly talking about the data object only in the second half of the sentence.